table_detection/table_neighbour[] neighbour_arr